api/src/service/cache2.ts

```diff
@@ -443,20 +444,25 @@ function addEventsToCache(cache: Cache2, streamName: string, newEvents: Business
       break;
 
     default:
-      // Do nothing, becaue informations will be reflected in aggregates
+      // Do nothing, because informations will be reflected in aggregates
       break;
   }
 }
 
 export function updateAggregates(ctx: Ctx, cache: Cache2, newEvents: BusinessEvent[]) {
-  // we ignore the errors
-  const { projects } = sourceProjects(ctx, newEvents, cache.cachedProjects);
+  const { projects, errors: pErrors = [] } = sourceProjects(ctx, newEvents, cache.cachedProjects);
+  if (!isEmpty(pErrors)) logger.warn("sourceProject caused error: ", pErrors);
```

Comment:
if there is an ill-formatted event in the history (for whatever reason), there will be a _warn_ printed _each time the uggregates are built up_. This is most likely not what we want here. Instead, we should either log `debug` level instead, or ignore the errors here altogether (like we did so far).